WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -747,6 +753,10 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
             "mentions": post.blog.isAccessibleThroughWPCom()
         ]
     }
+
+    func gutenbergEditorTheme() -> GutenbergEditorTheme? {
```

Comment:
Could this method move to the extension also?

Reply:
Yup I can move that. My thought here was that since it's a required method of `GutenbergBridgeDataSource` that it belonged with that extension, but I can just leave a comment on the method to help signify that.

Reply:
This has been moved now.